HandheldCompanion/Controls/GameView.xaml

```diff
@@ -0,0 +1,293 @@
+                            Command="{Binding RefreshGameIsRunningCommand}"
+                            ToolTip="Check Is Running"
+                            Content="Is Running?"
+                            Style="{DynamicResource AccentButtonStyle}">
+                        </Button>
+
+                        <Image
+                            Grid.Row="4"
+                            Grid.Column="0"
+                            Width="24"
+                            Height="24"
+                            HorizontalAlignment="Left"
+                            Source="{Binding IsRunningLogo}" />
+                    </Grid>
+                </Grid>
+
+            </ui:SimpleStackPanel>
+            <!--<TextBlock
+                Grid.Column="0"
+                Grid.Row="1"
+                Padding="10,10,10,10"
+                VerticalAlignment="Stretch"
+                TextWrapping="WrapWithOverflow"
+                HorizontalAlignment="Stretch"
+                TextAlignment="Left"
+                Text="All Games" />-->
+        </ui:SimpleStackPanel>
+        <ListBox 
+            Grid.Row="0"
+            d:Width="Auto"
+            d:Height="Auto"
+            Padding="0,10,0,20"
+            BorderThickness="0,0,0,0"
+            Foreground="{StaticResource PrimaryForegroundColor}"
+            ItemsSource="{Binding Games}" 
+            SelectedItem="{Binding SelectedGame,Mode=TwoWay}">
+            <ListBox.ItemsPanel>
+                <ItemsPanelTemplate>
+                    <WrapPanel />
+                </ItemsPanelTemplate>
+            </ListBox.ItemsPanel>
+            <ListBox.ItemContainerStyle>
+                <Style TargetType="ListBoxItem">
+                    <Setter Property="Template">
+                        <Setter.Value>
+                            <ControlTemplate TargetType="ListBoxItem">
+                                <Border
+                                    Background="{TemplateBinding Background}"
+                                    BorderThickness="4 0 0 0" 
+                                    BorderBrush="Transparent"
+                                    Cursor="Hand">
+                                    <ContentPresenter
+                                    HorizontalAlignment="Left"
+                                    VerticalAlignment="Center"
+                                    TextBlock.Foreground="{TemplateBinding Foreground}" />
+                                </Border>
+                            </ControlTemplate>
+                        </Setter.Value>
+                    </Setter>
+
+                    <Style.Triggers>
+                        <Trigger Property="IsMouseOver" Value="True">
+                            <Setter Property="Background" Value="{StaticResource ThirdBackgroundColor}" />
+                            <Setter Property="Foreground" Value="{StaticResource ThirdForegroundColor}" />
+                            <Setter Property="Template">
```

Comment:
Not a fan of all this Template manipulation through setters.
Why is the Template not set once using ListBox.ItemTemplate and having the Style setters/triggers applied there on the Border?